newdemojail.yaml

```
@@ -0,0 +1,772 @@
+entityDefinitions:
+  inmate:
+    fqn: "general.person"
+    entitySetName: "DemoPerson"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "SubjectIdentification"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "LastName"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "FirstName"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "MiddleName"
+      nc.PersonBirthDate:
+        type: "nc.PersonBirthDate"
+        column: "BirthDate"
+      nc.PersonSex:
+        type: "nc.PersonSex"
+        column: "Sex"
+      nc.SSN:
+        type: "nc.SSN"
+        column: "SocialSecurityNumber"
+      nc.PersonRace:
+        type: "nc.PersonRace"
+        column: "Race"
+      nc.PersonEthnicity:
+        type: "nc.PersonEthnicity"
+        column: "Ethnicity"
+    name: "inmate"
+  persondetails:
+    fqn: "ol.persondetails"
+    entitySetName: "DemoPersondetails"
```

```suggestion
    entitySetName: "DemoInmateDetails"
```

This also already exists. Please check for existing entity sets for all of these! Most already exist and duplicating would get very messy.

---

I am fixing to integrate into existing entity sets - except those prefixed with 'oldemoorgreentry'

---

Ones that existed already which I will be writing into:  DemoInmates, DemoInmateDetails,  DemoFacilities, DemoJailStays, DemoTimeServed, DemoCourtCases, DemoHearings, DemoLocations, DemoSentences,  DemoOfficers